From ec48ff093fd828b875d017b5e4f460b7cd365bac Mon Sep 17 00:00:00 2001
From: plmqazwsx <rkadam202@gmail.com>
Date: Sat, 9 Mar 2019 23:23:03 +0530
Subject: [PATCH] Testing patch link

Commit message here.
How can I add patch link here before generating a patch itself???
I'm following contribution.png sequence.
---
 my_git_commands_list | 37 -------------------------------------
 1 file changed, 37 deletions(-)

diff --git a/my_git_commands_list b/my_git_commands_list
index 403bd46..873bf00 100644
--- a/my_git_commands_list
+++ b/my_git_commands_list
@@ -1,19 +1,3 @@
-My list of git commands for quick look before I get used to them
-==========================================================================
-
-1. Start your own Repository
->> git init
-
-2.Download existing Repository
->> git clone https://github.com/cooperka/emoji-commit-messages.git
-
-3. Enter downloaded Repo and check status
->> cd emoji-commit-messages
->> git status
-
-4. Create new Local Checkpoint/Reference/File>SaveAs
->> git checkout -b my-new-feature
-
 5. Make changes in Repo
 >> echo “This is a cool new file” > my-file.txt
 >> git status
@@ -37,27 +21,6 @@ git add <files>
 or
 git add --all
 
-10. Status check after git add and changes2 will show new changes2 as "--newchanges2-in-red"
-git status
-
-11 Submit locally and also enter a message explaining new updates to codebase
-git commit
-or
-git commit -m "Message"
-
-12. Upload to github URL/remote Repo (called origin)
-Head is for git what current is for linux kernel. It gives latest checkpoint.
-git push origin <branch-name>
-or 
-git push origin HEAD
-
-13. Download branch
-Diff between pull and clone
-git pull <branch-name>
-
-14. Update local repo with detail about other branches at origin
-git fetch
-
 15. Merge
 git merge <other-branch-name>
 or
-- 
2.7.4

